packages/editor-ui/src/views/NodeView.vue

```diff
@@ -165,9 +170,27 @@ import {
 	XYPositon,
 	IPushDataExecutionFinished,
 	ITag,
+	IWorkflowTemplate,
 } from '../Interface';
 import { mapGetters } from 'vuex';
 
+const NODE_SIZE = 100;
+const DEFAULT_START_POSITION_X = 250;
+const DEFAULT_START_POSITION_Y = 300;
+const HEADER_HEIGHT = 65;
+const SIDEBAR_WIDTH = 65;
```

Comment:
since these don't necessarily need to be reused anywhere, I am reluctant to move them to a constants file